tests/test_build_latex.py

```diff
@@ -297,3 +297,16 @@ def test_contentsname_with_language_ja(app, status, warning):
     print(status.getvalue())
     print(warning.getvalue())
     assert '\\renewcommand{\\contentsname}{Table of content}' in result
+
+
+@with_app(buildername='latex', testroot='image')
+def test_image_in_section(app, status, warning):
+    app.builder.build_all()
+    result = (app.outdir / 'Python.tex').text(encoding='utf8')
+    print(result)
+    print(status.getvalue())
+    print(warning.getvalue())
```

Comment:
Ugh, didn't see that.  It all seems to have started in 8270575a9 — for no good reason.  If you care you can remove the other print statements as well (although I'd leave the one in `test_latex` in, since it's obviously only ever executed in case of test failure.)